src/check_fastq.py

```diff
@@ -27,6 +27,7 @@ def main():
         keys = ['s3://hca-util-upload-area/' + str(s3_object.key) for s3_object in my_bucket.objects.all()]
```

Comment:
A couple of comments because this is the first time I see this script:

- L12: Just a small thing, but they are upload areas UUIDs, not submission UUIDs
- L12: the argument name `--uuids` doesn't seem super appropriate since it's searching for a file path
- L12/13: remember to add short arguments! (e.g. -u), makes life much easier if we need to use this often
- It's nice to have an option to include a UUID file in case we need extensive checking, but maybe having another option to specify 1 uuid as this is what most likely will happen?
- L20/21: Reading this file with pandas may be a little bit of an overkill since we don't expect more than one field? I guess it does make it easier to read the code
- L25/26: declaring the s3 variable should only be done once, it's not needed per uuid as neither the bucket nor the credentials change. I suggest moving these 2 lines to before `for uuid in uuids:`